src/backend/services/request_validators.py

```diff
@@ -36,47 +33,43 @@ def validate_deployment_model(deployment: str, model: str, session: DBSessionDep
         HTTPException: If the deployment and model are not compatible
 
     """
-    deployment_db = deployment_crud.get_deployment_by_name(session, deployment)
-    if not deployment_db:
-        deployment_db = deployment_crud.get_deployment(session, deployment)
-
-    # Check deployment config settings availability
-    deployment_config = find_config_by_deployment_id(deployment)
-    if not deployment_config:
-        deployment_config = find_config_by_deployment_name(deployment)
-    if not deployment_config:
+    found = deployment_service.get_deployment_definition_by_name(session, deployment)
+    if not found:
+        found = deployment_service.get_deployment_definition(session, deployment)
+    if not found:
         raise HTTPException(
             status_code=400,
             detail=f"Deployment {deployment} not found or is not available in the Database.",
         )
 
-    if not deployment_db:
-        deployment_db = deployment_crud.create_deployment_by_config(session, deployment_config)
```

Comment:
Do we no longer need the functionality to create a deployment and model in the database if they were somehow deleted or a new type was added to the configuration?